index.html

```diff
@@ -46,17 +46,17 @@
   </head>
   <body>
     <div class="card">
-      <img src="my-profile-picture.png" alt="Girlcoder" style="width: 100%" />
-      <h1>Zandile Doe</h1>
-      <p class="title">Software developer</p>
-      <p>Girlcode Bootcamp</p>
+      <img src="/Natasha.jpg" alt="Natasha" style="width: 100%" />
+      <h1>Natasha Chitongo</h1>
+      <p class="title">Software Developer</p>
+      <p>Girlcode Software Development Learnership</p>
       <div style="margin: 24px 0">
-        <a href="https://twitter.com/GirlCode_za"><i class="fa fa-twitter"></i></a>
-        <a href="https://www.linkedin.com/company/girlcodeza"><i class="fa fa-linkedin"></i></a>
-        <a href="https://github.com/GirlCodeZA"><i class="fa fa-github"></i></a>
+        <a href="https://twitter.com"><i class="fa fa-twitter"></i></a>
+        <a href="https://www.linkedin.com"><i class="fa fa-linkedin"></i></a>
+        <a href="https://github.com"><i class="fa fa-github"></i></a>
       </div>
       <p>
-        <a href="mailto:girlcode@gmail.com"><button>Contact</button></a>
+        <a href="mailto:tashachitongo@gmail.com"><button>Contact</button></a>
```

Comment:
This file should only be modify in your repo https://github.com/natashac18/natashac18.github.io